### PR TITLE
Desktop: Resolves #10529 Korean translation error for new sub-notebook

### LINKED ISSUE #10529: Korean translation error
### Operating system

Windows

### Joplin version

2.14.22

### Desktop version info

Joplin 2.14.22 (prod, win32)

클라이언트 ID: ded66487635f4b7c932e20caaae4ce3e
동기화 버전: 3
프로파일 버전: 46
키체인 지원: 예

개정판: e579eb9

Backup: 1.4.0
Draw.io: 2.2.0
Email Note: 1.2.2
Freehand Drawing: 2.7.1
Markdown table calculations: 1.0.5
Markdown Table: Sortable: 1.1.0
Metis: 0.1.5
Rich Markdown: 0.13.3
Slash Commands: Datetime & More: 1.3.1

### Current behaviour

ko : 한국어 기반의 joplin 프로그램에서 "파일" 버튼을 클릭하고 3번쨰와 4번째의 항목이 번역이 잘못되었음
"새 노트북" 과 "하위 노트북" 이라고 번역해야되지만 번역이 정상적이지 않고 둘다 새 노트북으로 번역되어있음

en : When I clicked the "File" button in the Korean-based joplin program, the 3rd and 4th items were mistranslated.
It should be translated as new notebook and sub-notebook, but the translation is not normal and both are translated as new notebook.

en gui
image

ko gui
image

### Expected behaviour

한국어 기반의 gui의 메뉴에서 노트북 추가에 대한 ui의 번역오류를 수정해주세요

Please correct the translation error in the UI for adding a laptop in the menu of the Korean-based GUI.

### Logs

_No response_

### PR DESCRIPTION
**Pull request by tushar-coin**
Desktop: Resolves #10529 Korean translation error for new sub-notebook

Replaced the translation of new notebook with new sub-notebook in korean in ko.json resource file for desktop

### CODE REVIEW

**Comment by tushar-coin** [comment]
hey moderators would love to have your say on the edit

**Comment by tushar-coin** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by roman-r-m** [comment]
I'm pretty sure this file is automatically generated and must not be edited manually.
Read the docs on how to edit translations.

**Comment by tushar-coin** [comment]
hey @roman-r-m currently i am really new to this but tried taking reference from old prs and made new commits please review them

**Comment by roman-r-m** [comment]
I don't speak Korean, so can't check. Google translates it as "new sub note".
@doyun0202 can you pls check if the translation is correct.

**Comment by tessus** [comment]
btw, did you use poedit? Also, there is an additional empty line that isn't needed.

File: packages/tools/locales/ko.po [hunk 1]
```diff
@@ -3179,7 +3179,8 @@ msgstr "새 \"%s\" 노트북이 생성되고 \"%s\" 파일을 여기에 가져
 
 #: packages/app-desktop/gui/MainScreen/commands/newSubFolder.ts:6
 msgid "New sub-notebook"
-msgstr "새 노트북"
+msgstr "새 서브 노트"
+
```
Comment (tessus) [inline packages/tools/locales/ko.po]: why this empty line: 3183

**Comment by doyun0202** [comment]
@roman-r-m The "새 하위 노트" is suitable for Korean. 
@tessus The "새 서브 노트" is not suitable in Korean. 
So you have to translate it like this. 
 new sub notebook ->> 새 하위 노트북
new notebook ->> 새 노트북

**Comment by tessus** [comment]
@doyun0202 Thanks for your help. (Btw, I didn't translate it. My Korean is very basic thus I would not dare trying to. ;-)) I find it very interesting that Korean uses a lot of English load words like 노트북

@tushar-coin can you please make the changes accordingly.

File: packages/tools/locales/ko.po (hunk 1 above)
Comment (tushar-coin) [reply packages/tools/locales/ko.po]: a mistake while trying to check something will remove in the next commit

**Comment by tushar-coin** [comment]
@tessus on it

**Comment by tushar-coin** [comment]
> btw, did you use poedit? Also, there is an additional empty line that isn't needed.

i didn't also i am not able to understand the usecase of poedit here

**Comment by tessus** [comment]
You should always use Poedit, because manually hacking the file can lead to a broken translation file.

Also, I am not sure whether you read the other comments, but your translation is wrong. Additionally one is missing.

**Comment by tessus** [comment]
I am sorry, but I am closing this. I've pushed to correct translation in df8c7fd31

**Pull request closed without merging**